spec/models/download_spec.rb

```diff
@@ -53,7 +53,7 @@
       expect(Download.new(work).file_name).to eq("123456789-123456789-1234")
 
       work.title = "123456789 123456789 123456789"
-      expect(Download.new(work).file_name).to eq("123456789 123456789")
+      expect(Download.new(work).file_name).to eq("123456789_123456789")
     end
```

Comment:
Maybe an additional test for the edge case of a work title having several consecutive spaces like `A         mystery`?